### PR TITLE
add: testcontainers-python

### PR DESCRIPTION
**Pull request by YYYasin19**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by YYYasin19** [comment]
@conda-forge/help-python, ready for review!

File: recipes/testcontainers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    - neo4j-python-driver *
+    - opensearch-py *
+    - oracledb *
+    - pika *
+    - redis-py *
+    - selenium *
+    - weaviate-client >=4.5.4,<5.0.0
+    - chromadb-client *
+    - qdrant-client *
+    - bcrypt *
```
Comment (xhochy) [inline recipes/testcontainers/meta.yaml:55]: Please delete this as this will otherwise make coinstalling this package with something else really hard.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/testcontainers/meta.yaml (hunk 1 above)
Comment (YYYasin19) [reply recipes/testcontainers/meta.yaml:55]: Even the ones where a concrete version pinning is declared, e.g. `weaviate-client`?

Comment (YYYasin19) [reply recipes/testcontainers/meta.yaml:55]: Even the ones with a concrete version pinning, e.g. weaviate-client?

Comment (YYYasin19) [reply recipes/testcontainers/meta.yaml:55]: And does having `*` as constraint have an effect? I thought `run_constrained` meant that only WHEN the package is installed, the specific version (here `*`) is needed

Comment (xhochy) [reply recipes/testcontainers/meta.yaml]: All of them please. This is a wrong implementation of optional dependencies in grayskull

**Review by xhochy — APPROVED** [review]

**Pull request merged**